index.html

```diff
@@ -160,7 +160,7 @@ <h3>Multi-Post Stories</h3>
           </div>
         </article>
       </section>
-      <section class="container contact">
+      <section class="container contact" id="contact">
```

Comment:
[OPTIONAL]

- As the `mobile menu` milestone doesn't deal with the contact form that's why I'm making it optional but take it as a highly recommended suggestion and fix it before merging. The form input is overlapping the text in all the resolutions. Please fix it. Below is the screenshot.

![Screenshot from 2021-08-03 02-55-05](https://user-images.githubusercontent.com/22434660/127926292-e9ca3d57-555a-4e9f-8861-bbb604ed8946.png)